modules/retail/pages/retail-best-practices.adoc

```diff
@@ -1,75 +1,64 @@
+This flexibility allows for testing of new images or configurations. 
 
-* Build New Image
-
-  * Use Image Sync formula to synchronize the image to all Branch Servers
-
-* Create a Test Group
+//To ensure a smooth deployment, follow these steps:
+.Procedure: Smooth image deployment
 
+. Build new image.
+    Use image synchronization formula to synchronize the image to all Branch Servers.
+. Create a test group.
   * Name the group, for example, [literal]``TEST:POSVendor-Terminal``. The name can be chosen freely.
-  * Add the {saltboot} formula to this group, replicating the configuration from the original HWTYPE group, except for the image version. Use the name and version of the new image.
-
-* Move Selected Terminals
-
+  * Add the {saltboot} formula to this group, replicating the configuration from the original [literal]``HWTYPE`` group, except for the image version. 
+    Use the name and version of the new image.
+. Move selected terminals.
   * Identify the terminals you want to test the new image on.
-  * Transfer these terminals from the group [literal]``HWTYPE:POSVendor-Terminal`` group to the newly created group [literal]``TEST:POSVendor-Terminal``. It is recommended to use System Set Manager for this task.
-
-* Reboot Terminals
-
-  * Power on the selected POS terminals. They should boot the new image.
-
-* Verify and Evaluate
-
+  * Transfer these terminals from the group [literal]``HWTYPE:POSVendor-Terminal`` group to the newly created group [literal]``TEST:POSVendor-Terminal``. 
+    It is recommended to use System Set Manager for this task.
+. Reboot terminals.
+  * Power on the selected POS terminals. 
+    They should boot the new image.
+. Verify and evaluate.
   * Monitor the terminals in the [literal]``TEST:POSVendor-Terminal`` group.
   * Check if the new image performs as expected and if there are any issues or errors encountered.
-
-* Update HWTYPE Group
-
+. Update [literal]``HWTYPE`` group.
   * Once you have confirmed that the new image works correctly, the [literal]``HWTYPE`` group can be updated.
   * Modify the image version in the [literal]``HWTYPE:POSVendor-Terminal`` group to match the tested and approved version.
   * This ensures that all other terminals receive the new image.
 
 
-=== Controlling Image Versions
+=== Controlling image versions
 
 By default, {saltboot} formula selects the specified image name and version or selects the highest one if the version is not specified.
 
-There are multiple methods to control the image version within the HWTYPE and TEST groups:
+There are multiple methods to control the image version within the [literal]``HWTYPE`` and [literal]``TEST``` groups:
 
-* Specify Exact Version
-
-  * In both the HWTYPE and TEST groups, specify explicitly the exact version of the image to be deployed.
+.Procedure: Image control
```

Comment:
This paragraph is not a procedure, but rather list of options how can one control image deployment.
* specifying exact versions in A/B group created above
* using inactive flag to deactivate image
* using freeze flag to disable image autoupdate